docs/exploration-and-organization/collections.md

```diff
@@ -62,5 +64,10 @@ Note that you have to have Curate permission for the collection that you're movi
 
 You can add events to collections, and organize those events into timelines. See [Events and timelines](events-and-timelines.md).
 
+## Further reading
+
+- [Keeping your analytics organized](https://www.metabase.com/learn/administration/same-page)
```

Comment:
Could be interesting to link here, not sure if it is too in-the-weeds though:
https://www.metabase.com/learn/administration/multi-env#one-collection-per-environment